packages/terra-application-navigation/src/common/_PopupCount.jsx

```diff
@@ -8,13 +8,17 @@ import styles from './PopupCount.module.scss';
 const cx = classNames.bind(styles);
 
 const propTypes = {
+  /**
+   * Whether or not the count should be used as placeholder height.
```

Comment:
this description is a little confusing. Is this prop for toggling height or for hiding/showing the count value?